src/main/java/Entities/CollaborativeTask.java

```diff
@@ -0,0 +1,217 @@
+package Entities;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+public class CollaborativeTask extends Task implements Timeblockable{
+    private boolean recurring;
+    private String frequency;
```

Comment:
In my use case, I need to access the frequency so I would need a getter method for frequency.

Reply:
I created one for frequency and recurring just in case you need that as well.